cmd/testworkflow-init/main.go

```diff
@@ -104,13 +97,21 @@ func main() {
 			config["retryCount"] = os.Args[i+1]
 		case constants.ArgRetryUntil:
 			config["retryUntil"] = os.Args[i+1]
+		case constants.ArgToolkit, constants.ArgToolkitLong:
+			toolkit = true
+			i--
 		case constants.ArgDebug:
 			config["debug"] = os.Args[i+1]
 		default:
 			output.Failf(output.CodeInputError, "unknown parameter: %s", os.Args[i])
 		}
 	}
 
+	// Clean up unnecessary variables for non-toolkit containers
+	if !toolkit {
+		_ = os.Unsetenv("TK_REF")
```

Comment:
worth to log error

Reply:
I think that it should fail silently - it's not important for user